batch.py

```diff
@@ -488,11 +490,23 @@ def summarise_association_results(
     from multipy.fdr import qvalue
 
     logging.info('before glob (pv files) - summarise job')
-    existing_pv_files = list(to_path(pv_folder).glob('*_pvals.csv'))
+    storage_client = storage.Client()
+    bucket = get_config()['storage']['default']['default'].removeprefix('gs://')
```

Comment:
Maybe add a specific check for startswith gs:// to futureproof for azure / other file systems?

Reply:
I think there are bigger issues here if we move to non-google cloud, as this will be using an inappropriate client 😶